examples/braket_features/phase_estimation.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+from braket.circuits import Circuit
+
+
+def phase_estimation_circuit(n_precision_qubits: int, target_phase_in_resolution: int) -> Circuit:
+    """Phase estimation circuit. The circuit has 1 query qubit and `n_precision_qubits`
+    precision qubits. The resolution of phase estimation is `(1/2**n_precision_qubits)*2*pi`.
+    The circuit has `n_precision_qubits+1` qubits. To make benchmark easier, the target phase
+    is integer multiple of the phase resolution, `(fraction/2**n_precision_qubits)*2*pi`.
+    For example, if `n_precision_qubits=3` and `fraction=3`, the target phase is `(3/8)*2*pi`.
+
+    Args:
+        n_precision_qubits (int): Number of precision qubits.
+        target_phase_in_resolution (int): Number of fraction, (1/2**n_precision_qubits)*2*pi.
+            The oracle phase is (fraction/2**n_precision_qubits)*2*pi.
```

Comment:
Can we have the return statement in those doc-strings?

Reply:
updated